clustermesh/clustermesh.go

```diff
@@ -1901,3 +1886,35 @@ func (k *K8sClusterMesh) ConnectWithHelm(ctx context.Context) error {
 	k.Log("✅ Connected cluster %s and %s!", k.client.ClusterName(), remoteCluster.ClusterName())
 	return nil
 }
+
+func genClusterMeshConfig(aiLocal, aiRemote *accessInformation) map[string]interface{} {
+	// TODO (ajs): Support more than two clusters
+	return map[string]interface{}{
+		"clustermesh": map[string]interface{}{
+			"apiserver": map[string]interface{}{
+				"tls": map[string]interface{}{
+					"auto": map[string]interface{}{
+						"enabled":  true,
+						"method":   "cronJob",
+						"schedule": "0 0 1 */4 *",
+					},
+				},
+			},
+			"config": map[string]interface{}{
+				"enabled": true,
+				"clusters": []map[string]interface{}{
+					{
+						"name": aiLocal.ClusterName,
+						"ips":  []string{aiLocal.ServiceIPs[0]},
+						"port": aiLocal.ServicePort,
```

Comment:
Given that the TLS config is not specified, this does work only when the CA is shared. I would suggest to add a check at the beginning to prevent connecting two clusters when this condition does not hold, as it otherwise leads to a broken but difficult to troubleshoot setup.